paasta_tools/tron_tools.py

```diff
@@ -724,6 +745,357 @@ def get_projected_sa_volumes(self) -> Optional[List[ProjectedSAVolume]]:
         )
         return projected_volumes if projected_volumes else None
 
+    def get_kubernetes_service_account_name(self) -> Optional[str]:
```

Comment:
This is definitely a whole lot of repetition, but there's probably no other way for it without some huge refactor, so as long as CI is fine with it, I'm fine.